src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/UserListField.groovy

```diff
@@ -1,10 +1,9 @@
 package com.netgrif.application.engine.petrinet.domain.dataset
 
-class UserListField extends Field<List<String>> {
+class UserListField extends Field<UserListFieldValue> {
```

Comment:
As far as I know the user list field is analogous to the file list field.

As far as I know the value of the file list field is `List<FileFieldValue>`.

Why is this implemented differently? The `UserListFieldValue` class contains a List of FileFieldValue-s, but that changes how the field value is accessed in actions and makes it inconsistent.

```
fileListField: f.myFileListField,
userListField: f.myUserListField;

fileListField.value; // is a List
userListField.value.userValues; // is the list we expect
```

Reply:
The `FileListField` has a definition `class FileListField extends Field<FileListFieldValue>` and the `FileListFieldValue`contains set of `FileFieldValue`. The implementation in case of `UserListField` is the same.

Reply:
@renczesstefan is right. If you think this should be implemented differently please rise a feature request. Thank you.